e2e/tests/components/character-count/fixtures.ts

```diff
@@ -0,0 +1,10 @@
+import { expect, test as base } from '../../../fixtures'
+
+const test = base.extend({
+  getCharacterCountMessage: ({ page }, use) => {
+    // This needs to be more specific, as there is a second identical message just for screen readers
+    use((text: string) => page.getByText(text).filter({ has: page.locator('.govuk-character-count__message') }))
```

Comment:
The locator could equally be `page.locator('.govuk-character-count__message', { hasText: text }))`, but I thought using `getByText()` makes the logic and rationale easier to understand here.